index.dev.js

```diff
@@ -25,56 +26,59 @@ function runInsect(fmt, line) {
   return res;
 }
 
-// Handle command line arguments
-if (process.argv.length >= 4) {
-  usage();
-} else if (process.argv.length == 3) {
-  var arg = process.argv[2];
-  if (arg === "-h" || arg === "--help") {
+// Top-level await is not supported in Node 12 and earlier.
+(async function() {
+  if (process.env.INSECT_NO_RC !== "true") {
+    function processRcLine(line) {
+      var res = runInsect(Insect.fmtPlain, line);
+      // We really only care when it breaks
+      if (res && res.msgType === "error") {
+        console.error(res.msg);
+        process.exit(1);
+      }
+    }
+
+    var [util, lineReader] = await Promise.all([import("util"), import("line-reader")]);
+    var eachLine = util.promisify(lineReader.eachLine);
+
+    try {
+      await eachLine(path.join(xdgBasedir.config, "insect/insectrc"), processRcLine);
```

Comment:
Any suggestions for a filename other than `insectrc`? I personally think `insectrc` is good, but I'd like to hear opinions.

Reply:
Another option would be to use sth. like `.ins` as extension and call it `init.ins` or similar. But I'm fine with `insectrc`.

Reply:
Then let's stick to `insectrc`.